### PR TITLE
Add a note about .terraformignore to CLI workflow

### PR DESCRIPTION
**Pull request by pselle**
Depends on: release of .terraformignore support

### CODE REVIEW

File: content/source/docs/cloud/run/cli.html.md [hunk 1]
```diff
@@ -86,6 +86,12 @@ If you manage your Terraform configurations in self-contained repositories, the
 
 If you use a combined repository and [specify a working directory on workspaces](../workspaces/settings.html#terraform-working-directory), you can run Terraform from either the real working directory or from the root of the combined configuration directory. In both cases, Terraform will upload the entire combined configuration directory.
 
+## Excluding files from upload
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/cli.html.md]:
```suggestion
## Excluding Files from Upload
```

Title-case for headers.

File: content/source/docs/cloud/run/cli.html.md [hunk 2]
```diff
@@ -86,6 +86,12 @@ If you manage your Terraform configurations in self-contained repositories, the
...
+
+When using in the CLI-driven run workflow, an archive of your configuration directory is
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/cli.html.md]:
```suggestion
CLI-driven runs upload an archive of your configuration directory
```

File: content/source/docs/cloud/run/cli.html.md [hunk 3]
```diff
@@ -86,6 +86,12 @@ If you manage your Terraform configurations in self-contained repositories, the
...
+uploaded to Terraform Cloud. There may be files you would like to exclude from upload,
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/cli.html.md]:
```suggestion
to Terraform Cloud. If the directory contains files you want to exclude from upload,
```

File: content/source/docs/cloud/run/cli.html.md [hunk 4]
```diff
@@ -86,6 +86,12 @@ If you manage your Terraform configurations in self-contained repositories, the
...
+and you can do so by defining a [`.terraformignore` file in your configuration directory][remote].
```
Comment (nfagerlund) [inline content/source/docs/cloud/run/cli.html.md]:
```suggestion
you can do so by defining a [`.terraformignore` file in your configuration directory][remote].
```

File: content/source/docs/cloud/run/cli.html.md (hunk 1 above)
Comment (nfagerlund) [reply content/source/docs/cloud/run/cli.html.md]: (Sidenote: while I was second-guessing myself on that "from," I found this, which IMHO is excellent. https://titlecaseconverter.com/words-to-capitalize/?style=AP,CMOS,MLA,NYT,WP#from)

File: content/source/docs/cloud/run/cli.html.md (hunk 2 above)
Comment (pselle) [reply content/source/docs/cloud/run/cli.html.md]: Thanks, I was trying to follow/see if "run workflow" was some sort of terminology we'd decided to stick to, but I find it confusing. Grateful to simplify :)

**Comment by nfagerlund** [comment]
@pselle Oh yeah, good call, merging before release will save you a step! 👍🏼 

The website doesn't auto-deploy on merge, but it generally deploys a few times a day anyway just because some provider is always having a dot-z release. Manual deploys can be kicked on teamcity.

**Comment by nfagerlund** [comment]
Ah, and now I see it's already shipped 😅 ok, I'll just merge and deploy.

**Comment by pselle** [comment]
Thanks @nfagerlund!

**Pull request merged**